nsv13/code/modules/overmap/overmap.dm

```diff
@@ -168,7 +169,7 @@
 	vector_overlay.icon = icon
 	vis_contents += vector_overlay
 	update_icon()
-	max_range = initial(weapon_range)+20 //Range of the maximum possible attack (torpedo)
+	max_range = 50 //Range of the maximum possible attack (torpedo) - Magic number pulled from the aether
```

Comment:
This number I have now noticed in multiple places can you make it a define please.

Reply:
Negative. This does not need a define for a fixed number that is a stop gap for reworking of weapon_range. I have tagged it as such.

Reply:
This is a temp bodge fix but I told him to do it until corvy can rework weapons